src/java.base/share/conf/security/java.security

```diff
@@ -642,8 +633,7 @@ jdk.disabled.namedCurves = secp112r1, secp112r2, secp128r1, secp128r2, \
 #
 #
 jdk.certpath.disabledAlgorithms=MD2, MD5, SHA1 jdkCA & usage TLSServer, \
-    RSA keySize < 1024, DSA keySize < 1024, EC keySize < 224, \
-    include jdk.disabled.namedCurves
+    RSA keySize < 1024, DSA keySize < 1024, EC keySize < 224
```

Comment:
jdk.disabled.namedCurves is commented out and I don't think it's good for every operation disabled algorithms call to check an empty property.  The description for the disabledAlgorithm properties say you have to use "include", so I don't think it is necessary to we keep it active..

Reply:
I just think this is an unnecessary behavior change. After all, the purpose of `jdk.disabled.namedCurves` is to be included in other disabledAlgorithms properties.

No strong opinion on this. Please decide yourself.

Reply:
I understand what you are saying.  The property only existed because there were so many curves that would have overwhelmed the disabledAlgorithms.  I also don't like making this a permanent addition to the disabledAlgorithm properties. It's possible we may remove the property in the future as it's likely unnecessary going forward.